omod/src/main/java/org/openmrs/web/dwr/DWRPatientService.java

```diff
@@ -466,6 +467,9 @@ public String addIdentifier(Integer patientId, String identifierType, String ide
 	}
 	
 	/**
+	 * Copied from OpenMRS core 1.12.x and modified the method to use {@link LegacyUIService#exitFromCare(Patient, Date, Concept)}
+	 * See https://github.com/openmrs/openmrs-core/blob/1.12.x/web/src/main/java/org/openmrs/web/dwr/DWRPatientService.java
```

Comment:
Same as above for using tags

Reply:
[Fixed](https://github.com/openmrs/openmrs-module-legacyui/pull/128/commits/9dd92c8a5560377daedb2e231c639e315fa177b8)